### ISSUE TITLE
Inference on custom dataset

### ISSUE THREAD

**Issue by kartikgupta2607** [SysCV/pcan#17]
Hey there, thank you for the great work!
Is there any inference script using which we can run the pre-trained model on custom video (without ground truth) and get the output for the MOTS task (preferably instance_id masks for each frame)? I tried using the demo script from MMTracking, but there were dependency issues for the versions of MMCV and MMDet while using PCAN.

**Comment by lkeab** [comment]
Hi, you can modify the dataloader [here](https://github.com/SysCV/pcan/blob/06416f1c96b7a86754828582d9a95b9ce0d327ba/pcan/apis/test.py#L23) to let it read image sequence only from the video directories w/o reading the corresponding GT labeling. Or you can generate fake annotation GT to mimick the current GT format with empty mask and box labels.
